installation/resources/watch-pods.yaml

```diff
@@ -48,7 +48,7 @@ metadata:
 spec:
   containers:
   - name: watch-pods
-    image: eu.gcr.io/kyma-project/develop/watch-pods:c3c08515
```

Comment:
`watch-pod` has been recently moved to test-infra. So, it doesn't have `0.8.0-rc1` tag. Either revert this or release it from test-infra.

Reply:
done